crates/polars-stream/src/physical_plan/mod.rs

```diff
@@ -25,16 +25,23 @@ pub enum PhysNode {
     InMemorySource {
         df: Arc<DataFrame>,
     },
+    Select {
+        input: PhysNodeKey,
+        selectors: Vec<ExprIR>,
+        extend_original: bool,
+        schema: Arc<Schema>,
+    },
     Filter {
         input: PhysNodeKey,
         predicate: ExprIR,
     },
     SimpleProjection {
         input: PhysNodeKey,
-        schema: SchemaRef,
```

Comment:
Why not `SchemaRef`?  It's a bit easier to write. :)

Reply:
I think it's barely easier to write, but makes it harder to understand what's going on. I'd rather know directly that I have an `Arc` to a `Schema`.